lib/selenium-webdriver/webdriver.js

```diff
@@ -236,7 +236,7 @@ webdriver.WebDriver.prototype.executeAsyncScript = (script, var_args) => {};
  * button.click();
  * browser.call(logText, counter);
  *
- * @param {function(...): (T|promise.Promise<T>)} fn The function to
+ * @param {function(...*): (T|promise.Promise<T>)} fn The function to
```

Comment:
If this change is not related to the banner update, I'd recommend reverting back.

Reply:
Happy to do so! Yeah, I'm a little concerned about the build process to actually deploy this as there were a few docs / build scripts that felt like they may be half-updated. I was able to get it to build, but it required updating the `dgeni` version for a bug fix (and thus where this came in)